package.json

```diff
@@ -19,18 +19,16 @@
     },
     "devDependencies": {
         "aws-sdk-mock": "^2.0.0",
-        "babel-cli": "^6.26.0",
         "babel-eslint": "^8.2.2",
```

Comment:
since we removed this custom parser from `eslint` config, it's ok to remove this package altogether